docs/UserGuide.md

```diff
@@ -205,10 +207,14 @@ Examples: <br>
 
 Deletes the specified person from the address book.
 
-Format: `delete INDEX`
+Format: `delete INDICES`
 
-* Deletes the person at the specified `INDEX`.
+* Deletes the person(s) at the specified `INDEX` or range of indices.
 * The index refers to the index number shown in the **last displayed person list**.
+* `INDICES` can be a single number (e.g., `2`) or a closed range (e.g., `5-9`), separated by spaces (e.g., `1 2 3 5-9`).
+  * **Note:** In a closed range, there must be no spaces between the numbers and the hyphen (e.g., `5-9` is correct, 
+    but `5 - 9` or `5 -9` or `5- 9` is invalid).
+* `INDICES` should not be empty; at least one `INDEX` or `range` must be provided.
```

Comment:
I don't think `INDEX` or `range` should be formatted as such as there is no other mention of it. The closest would be the "single number" and "closed range" above.

I would think writing `INDICES` should not be empty is sufficient

Reply:
ok changes pushed